features/src/main/feature/camel-features.xml

```diff
@@ -560,10 +563,26 @@
         <bundle>mvn:org.apache.camel.karaf/camel-azure-servicebus/${project.version}</bundle>
     </feature>
     <feature name='camel-azure-storage-blob' version='${project.version}' start-level='50'>
+        <feature prerequisite="true">spifly</feature>
```

Comment:
Should not be needed anymore as it is added by the plugin. Have you tried without it?